image-loader/conf/org/apache/tika/mime/custom-mimetypes.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info>
+  <mime-type type="image/x-raw-canon">
+    <_comment>Canon raw image</_comment>
+    <magic priority="50">
```

Comment:
nit: This has the same priority as tiff files which I think should also match this magic. Should we increase this priority to make sure that whatever tika uses to tiebreak doesn't accidentally switch back to identifying these as tiffs?

Reply:
Sounds reasonable.  I think custom-mimetypes have higher priority already, but no harm in increasing this.